src/modules/missionHelper/i18n/en_US.json

```diff
@@ -57,7 +57,7 @@
 			"k9": "K-9",
 			"large_fire_boat": "Large Fireboat | Large Fireboats",
 			"large_rescue_boat": "Large Rescueboat | Large Rescueboats",
-			"mobile_air_vehicles": "Mobile Air Vehicle | Mobile Air Vehicles",
+			"mobile_air_vehicles": "Mobile Air Unit | Mobile Air Unit",
```

Comment:
Add the "s" in Plural => "Units"

Reply:
nöp i checked this ingame :D so is right